src/main/java/com/naijagov/naijagov/config/MvcConfig.java

```diff
@@ -8,6 +8,6 @@
     
     public void addViewControllers(ViewControllerRegistry registry){
         registry.addViewController("/pollingunitsreact").setViewName("pollingunitsreact");
-        registry.addViewController("/pollingunits").setViewName("pollingunits");
+        registry.addViewController("/pollingunitsmaterial").setViewName("pollingunitsmaterial");

```

Comment:
I strongly recommend using camel-case naming convention or some other convention. The was it is now would be a bit difficult to decipher

Reply:
Fixed in commit [bc618b2](a1e795154d7813c8448dfd349695fe3b0ddcf088)